app/ledger/n3NeonLedger.js

```diff
@@ -14,6 +22,32 @@ export const MESSAGES = {
     'Error parsing transaction. Make sure your NEO3 Ledger app version is up to date',
 }
 
+/**
+ * Evaluates Transport Error thrown and rewrite the error message to be more user friendly.
+ * @param {Error} err
+ * @return {Error}
+ */
+const evalTransportError = error => {
```

Comment:
Any specific reason why we moved to this back in? It's a duplication of https://github.com/CityOfZion/neon-js/blob/820e20d3160e46d019c3f65e84d3f9270ffa1588/packages/neon-ledger/src/ErrorCode.ts#L38

but with wrong status words.